scripts/xe-reset-networking

```diff
@@ -184,6 +187,8 @@ Type 'no' to cancel.
 		f = file(management_conf, 'w')
```

Comment:
I agree with you, but I think that in that case we should rewrite the whole script to use the template string.
I think it's a too big change, and it's probably safer to minimize the change and keep it consistent as it is now.